files/autostop/stop.sh

```diff
@@ -6,4 +6,8 @@ if isTrue "${DEBUG_AUTOSTOP}"; then
 fi
 
 logAutostopAction "Stopping Java process"
-kill -SIGTERM 1
+if isTrue "${AUTOSTOP_PKILL_USE_SUDO}"; then
```

Comment:
I'm am gradually working on a documentation tool extract variables and defaults, so it'll be great if this is

```suggestion
if isTrue "${AUTOSTOP_PKILL_USE_SUDO:-false}"; then
```